src/components/CustomerQuota/NoQuota/TransactionTitle.tsx

```diff
@@ -1,20 +1,27 @@
 import React, { FunctionComponent } from "react";
-import { formatDistance } from "date-fns";
 import { AppText } from "../../Layout/AppText";
 import { sharedStyles } from "../sharedStyles";
-import { formatDateTime, formatDate } from "../../../utils/dateTimeFormatter";
+import i18n from "i18n-js";
+import {
+  formatDateTime,
+  formatDate,
+  formatTimeDifference
+} from "../../../utils/dateTimeFormatter";
 
 export const DistantTransactionTitle: FunctionComponent<{
   transactionTime: Date;
   toggleTimeSensitiveTitle: boolean;
 }> = ({ transactionTime, toggleTimeSensitiveTitle }) => (
   <>
-    <AppText style={sharedStyles.statusTitle}>Limit reached on </AppText>
     <AppText style={sharedStyles.statusTitle}>
-      {formatDateTime(transactionTime)}
+      {i18n.t("checkoutSuccessScreen.limitReached", {
+        dateTime: formatDateTime(transactionTime)
+      })}
```

Comment:
Looks like instead of 2 `AppText` components we are only doing one. Any impact to visual?

Reply:
Same for other titles in this file

Reply:
no change in visual! 
![photo6292025229651847856](https://user-images.githubusercontent.com/38589870/93860225-54e78b80-fcf1-11ea-8a2f-44dd9e61586d.jpg)
![photo6292025229651847855](https://user-images.githubusercontent.com/38589870/93860338-8a8c7480-fcf1-11ea-97a3-374ec38dbe07.jpg)